pkg/interactive/validation.go

```diff
@@ -101,3 +100,22 @@ func RegExp(restr string) Validator {
 		return fmt.Errorf("can only validate strings, got %v", val)
 	}
 }
+
+func RegExpBoolean(restr string) Validator {
+	re := regexp.MustCompile(restr)
+	return func(val interface{}) error {
+		if boolVal, ok := val.(bool); ok {
+			var val string
+			if boolVal {
+				val = "true"
+			} else {
+				val = "false"
+			}
+			if !re.MatchString(val) {
+				return fmt.Errorf("%s does not match regular expression %s", val, re.String())
```

Comment:
Since you have a function specifically for this, why not have an error that represents the boolean nature: `"%s can only be true or false"` or something along those lines.

Reply:
Yeah I played around with that, I am unsure as it is a boolean it can only be true or false anyway. The regular expression will only be forcing it to be explicitly one of them. In this case it might be better to show the expression. 

We will soon have the custom validation error messages, maybe we can refactor this to return that error instead :thinking: